shell/components/form/KeyValue.vue

```diff
@@ -598,6 +598,7 @@ export default {
     <div
       class="kv-container"
       role="grid"
+      :aria-label="title || t('generic.ariaLabel.keyValue')"
       :aria-rowcount="rows.length"
       :aria-colcount="extraColumns.length + 2"
```

Comment:
I think that we need to address this to ensure that the column counts are accurate. Right now, we have situations where the column is called out as "Column3" but there is no column 1 or 2.

Reply:
I've created an issue to tackle this separately: https://github.com/rancher/dashboard/issues/14439